gnucash/gnucash-core-app.cpp

```diff
@@ -219,6 +219,13 @@ Gnucash::CoreApp::parse_command_line (int argc, char **argv)
 {
     try
     {
+    // don't process args on and after "--"
+    for (int i = 0; i < argc; ++i)
+        if (!g_strcmp0 (argv[i], "--"))
+        {
+            argc = i;
+            break;
```

Comment:
No need for `break` because immediately after the previous statement, the for loop exit condition will be satisfied😭